### DISCUSSION TITLE
Again empty space left and right

### DISCUSSION THREAD

**Discussion by FanIT** [Live-Charts/LiveCharts2#156] [Q&A] [answered]
Hi, Beto. I opened [issue](https://github.com/beto-rodriguez/LiveCharts2/issues/36) a few months ago about it, but now i have updated LiveChart to beta 60 and again get empty spaces left and right.
How to hide the empty spaces agains?
View:
```
<lvc:CartesianChart Series="{Binding ChartModel.Series}" YAxes="{Binding ChartModel.YAxis}"
                            XAxes="{Binding ChartModel.XAxis}" BorderBrush="Red" BorderThickness="2"/>
```
Model:
```
_lineSeries = new LineSeries<ObservablePoint>
            {
                Values = _observablePoints,
                Stroke = null,
                GeometryFill = null,
                GeometryStroke = null,
                DataPadding = new PointF(0, 1)
            };
```
![Chart](https://user-images.githubusercontent.com/20283004/130424992-09da8f94-f093-4d8f-8a25-7704e6b51d62.PNG)

**Comment by beto-rodriguez** [comment, marked as answer]
My bad, this was broke when fixing #128.

Currently in beta.60 this is not working property.

But I just committed bb84f58eded4fc4b44223ba1f3d0ef4a95d05f2f, it fixes this issue, there will be a couple of changes in the next release of the library due #158 and #128.

I added a sample that works with the new changes:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/ViewModelsSamples/Lines/Padding/ViewModel.cs

**Reply by FanIT** [reply]
Thank you. I will be wait the next release.
